apps/ios/GuideDogs/Code/Visual UI/Views/Onboarding/OnboardingWelcomeView.swift

```diff
@@ -61,14 +77,19 @@ struct OnboardingWelcomeView: View {
             AppContext.shared.eventProcessor.deactivateCustom()
         }
     }
-    
 }
 
+// Mocked UserSettings for demonstration purposes
+struct UserSettings {
+    static var alwaysShowLanguageSelection: Bool = false // User-configurable setting
+}
```

Comment:
If this is meant to be a user-configurable setting, this would need to be propagated to a settings view. But per #112 I think we just wanted to change the heuristic for when the language choice should be shown, not add another setting.